sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/models/SchemaProperties.java

```diff
@@ -18,16 +18,22 @@ public final class SchemaProperties {
 
     private final String id;
     private final SchemaFormat format;
+    private final String groupName;
+    private final String name;
 
     /**
      * Initializes a new instance.
      *
      * @param id The schema id.
      * @param format The type of schema, e.g. avro, json.
+     * @param groupName The schema group for this schema.
+     * @param name The name of the schema.
      */
-    public SchemaProperties(String id, SchemaFormat format) {
+    public SchemaProperties(String id, SchemaFormat format, String groupName, String name) {
```

Comment:
This is a breaking change. So, we'll have to deprecate this constructor and add another overload with these additional args. Since this is an output-only model, you can even consider using an accessor pattern to have private setters and not add a new constructor at all.